### DISCUSSION TITLE
Error on try run-test in AWS.

### DISCUSSION THREAD

**Discussion by pedropnaves** [artilleryio/artillery#1105] [Artillery]
Guys, I had one error on try run-test in aws. I create a cluster, pass the region and my test. I think that my mistake is because I dont send token. but how do this ? 
This is my ECS Fargate log

```
fatal error: An error occurred (404) when calling the HeadObject operation: Key "artillerypro.rc" does not exist
```

and this is the command i ran
```
artillery run-test   --cluster artillery-dev   --region us-east-1   --count 1   --launch-type ecs:fargate   user-get-status.yml
```

**Comment by pedropnaves** [comment]
My problem was resolved.

I had deployed my backend in another AWS account.

**Comment by robbiecloset** [comment]
I am encountering this issue as well. 

@pedropnaves can you elaborate on the solution you found? I have just a single AWS account to which everything is deployed at the moment.

**Comment by hassy** [comment]
That error message may be ignored, it has no effect on whether a test run succeeds or not.
